i18n/en.yaml

```diff
@@ -491,10 +491,16 @@ FALLOUT.TOOLTIPS.WeaponQuality.unreliable: When you make an attack with an Unrel
 FALLOUT.TOOLTIPS.WeaponQuality.unstable_radiation: This was never designed to be used as a handheld weapon. When attacking with a weapon, each complication you inflicts 2DC Radiation damage from radioactive feedback. A weapon may not have both Reliable and Unstable Radiation.
 FALLOUT.UI.Add: Add
 FALLOUT.UI.Addictions: Addictions
+FALLOUT.UI.Additionalamo: Additional damage dice from firerate
+FALLOUT.UI.Additionalmeledmg: Additional damage dice for party AP
+FALLOUT.UI.AdditionalmeledmgOverseer: Additional damage dice for overseer AP
```

Comment:
As mentioned in your other pull request, please try and keep keys readable. Going forward I'm trying to standardise the i18n key style to snake_case, avoiding unecessary abbreviations where possible